ios/src/TGMapViewController.mm

```diff
@@ -9,11 +9,13 @@
 
 #import "TGMapViewController.h"
 #include "platform_ios.h"
+#include "data/propertyItem.h"
 #include "tangram.h"
 
 @interface TGMapViewController ()
 
-@property (nullable, strong, nonatomic) EAGLContext *context;
+@property (nullable, nonatomic) NSString* scenePath;
```

Comment:
The problem is that even if we use `NSURL` from the public interface and store it here, we would only end up using `[NSURL absolutePath]` and convert this to a `char*` string for use in the Tangram API. Then, later on this path would get resolved by the iOS platform layer using the bundle resolution; where we would loose what this `NSURL` object has to offer for tangram.

Considering what `NSURL` can do and what we would end up using within this wrapper, it seems that we can keep on using `NSString`. This wouldn't stop the user from using it at a higher level (in the SDK for example).